deepchem/models/graph_models.py

```diff
@@ -1001,7 +1001,7 @@ def default_generator(self,
           batch_size=self.batch_size,
           deterministic=deterministic,
           pad_batches=pad_batches):
-        if y_b is not None and self.mode == 'classification':
+        if y_b is not None and self.mode == 'classification' and not(mode=='predict'):
```

Comment:
Yapf is failing on this PR. To fix, install yapf
```
pip install yapf==0.32
```
Then run
```
yapf -i <filename>
```
on each modified file